sdk/search/Azure.Search.Documents/src/Generated/IndexesRestClient.cs

```diff
@@ -32,7 +32,7 @@ internal partial class IndexesRestClient
         /// <param name="xMsClientRequestId"> The tracking ID sent with the request to help with debugging. </param>
         /// <param name="apiVersion"> Api Version. </param>
         /// <exception cref="ArgumentNullException"> <paramref name="clientDiagnostics"/>, <paramref name="pipeline"/>, <paramref name="endpoint"/> or <paramref name="apiVersion"/> is null. </exception>
-        public IndexesRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint, Guid? xMsClientRequestId = null, string apiVersion = "2024-11-01-preview")
+        public IndexesRestClient(ClientDiagnostics clientDiagnostics, HttpPipeline pipeline, string endpoint, Guid? xMsClientRequestId = null, string apiVersion = "2025-03-01-preview")
```

Comment:
We have to update this new preview version in the [SearchClientOptions](https://github.com/Azure/azure-sdk-for-net/blob/ff165e458a063f032dd3732d0fe58be5b8741fd2/sdk/search/Azure.Search.Documents/src/SearchClientOptions.cs#L44-L47) class.

Reply:
I'm aware. Just intended to do so in another PR. I'll have it updated in this one to make it faster.

Reply:
Done.